makefiles/variables.mk

```diff
@@ -66,33 +66,46 @@ GIT_STATUS := git status
 #- duplicated in "env.sh" for the env var with the same name
 NIMBLE_DIR := vendor/.nimble
 REPOS_DIR := vendor
+
 ifeq ($(OS), Windows_NT)
   PWD := pwd -W
+  EXE_SUFFIX := .exe
+  # available since Git 2.9.4
+  GIT_TIMESTAMP_ARG := --date=unix
 else
   PWD := pwd
+  EXE_SUFFIX :=
+  # available since Git 2.7.0
+  GIT_TIMESTAMP_ARG := --date=format-local:%s
 endif
+
+ifeq ($(shell uname), Darwin)
+  # md5sum - macOS is a special case
+  MD5SUM := md5 -r
```

Comment:
do we use this somewhere?

Reply:
https://github.com/status-im/nimbus-eth1/blob/de12fb3c9405d431e864db74a50034f113a83e73/Makefile#L103